samples/resources-gallery/web-app/build.gradle.kts

```diff
@@ -17,6 +17,7 @@ kotlin {
 
                 implementation(compose.html.core)
                 implementation(compose.runtime)
+                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.8.1-Beta")
```

Comment:
why this should be added?

Reply:
After upgrade sample with: kotlin 1.9.23 and compose multiplatform 1.6.1 for this block:
`
val color by remember(window) {
                    Testing.getGradientColors().first().getColorFlow(window)
                }.collectAsState(initial = MokoColor(0xFFAAFFFF))
`
I'm get error: idea can't identify collectAsState, after add coroutines i'm get success build and run

Reply:
I think need set stable version